src/editors/containers/TextEditor/components/SelectImageModal/hooks.js

```diff
@@ -31,15 +34,15 @@ export const displayList = ({ sortBy, searchString, images }) => module.filtered
 }).sort(sortFunctions[sortBy in sortKeys ? sortKeys[sortBy] : sortKeys.dateNewest]);
 
 export const imgListHooks = ({
-  fetchImages,
   setSelection,
   searchSortProps,
 }) => {
+  const dispatch = useDispatch();
   const [images, setImages] = module.state.images({});
   const [highlighted, setHighlighted] = module.state.highlighted(null);
 
   React.useEffect(() => {
-    fetchImages({ onSuccess: setImages });
```

Comment:
We're getting error "checkPropTypes.jss?a15c:20 Warning: Failed prop type: The prop `highlighted` is marked as required in `Gallery`, but its value is `null`."

Should we change the PropTypes check or init with ""?